src/common/KBaseButton.module.scss

```diff
@@ -0,0 +1,11 @@
+.button {
+  /*button styles*/
+  border-color: #000;
```

Comment:
Probably should use a kbase color from colors.scss (here and below)

Reply:
I might have gone too far in the other direction, but I think we'll want [classes like this](https://github.com/kbase/ui-refresh-test/pull/15/files#diff-6ea08c95a60c10153bc1d5ae1e330e1fa103cc61df85954a91f741359450a5b0R28) at some point anyway. Let me know what you think.